apps/nextjs/src/app/aila/help/index.tsx

```diff
@@ -28,11 +29,19 @@ const Help = () => {
 
   const searchParams = useSearchParams();
   const ailaId = searchParams.get("ailaId");
-
+  const demo = useDemoUser();
+  const marginTop = demo.isDemoUser ? "200px" : "125px";
   return (
     <>
       <Header />
-      <div className="mx-auto mt-30 h-[100vh] w-full max-w-[1280px] px-9">
+      <OakFlex
+        $maxWidth={"all-spacing-24"}
+        $ml="auto"
+        $mr="auto"
+        style={{ marginTop: marginTop }}
```

Comment:
Guessing there's a reason we can't use `$mt` here instead?

Reply:
Maximum is 80 in the  `oakSpaceBetweenTokens` prop - because it's a fixed header, I've had to increase the maximum here so that it's not obscured.